server/src/main/models/index.ts

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var fs: any        = require('fs');
+var path: any      = require('path');
+var Sequelize: any = require('sequelize');
+var basename: string  = path.basename(__filename);
+var env: string       = process.env.NODE_ENV || 'development';
+var config: any    = require(__dirname + '/..\..\..\src\main\config\config.js')[env];
+var db: any        = {};
```

Comment:
`const` or `let` since this is TS. (preferrably `const` in this case)

Reply:
Oh, also, `db` is of type `{[key: string]: any}`. I'm not sure what uses it and how, but it has the "sequelize" and "Sequelize" properties so those can be added to its type.